src/main/java/core/basesyntax/service/impl/StorageReductionServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.StorageService;
+import core.basesyntax.service.TransactionLog;
+
+public class StorageReductionServiceImpl implements StorageService {
+
+    @Override
+    public void actionToStorage(TransactionLog transactionLog, Storage storage) {
```

Comment:
You can make your storage static and access it directly. Or make it a dependency of this class, but it's better not to pass storage in method parameters